Content.Server/Speech/EntitySystems/SlurredSystem.cs

```diff
@@ -61,46 +61,62 @@ private string Accentuate(string message, float scale)
         // This is pretty much ported from TG.
         foreach (var character in message)
```

Comment:
Cleanup: consider pulling the loop contents out into its own function to drop the function's complexity down, and tidying up the if-elses in here. Generally it's hard to keep track of code like this.